src/blockchain/Pledges.js

```diff
@@ -277,12 +278,52 @@ class Pledges {
 
   _doTransfer(transferInfo) {
     const donations = this.app.service('donations');
-    const { donation, amount } = transferInfo;
+    const { toPledgeAdmin, toPledge, toPledgeId, delegate, intendedProject, donation, amount, ts } = transferInfo;
 
     if (donation.amount === amount) {
       // this is a complete pledge transfer
       const mutation = this._createDonationMutation(transferInfo);
 
+      if(mutation.status === 'committed') {
+        // send a receipt to the donor
+        Notifications.donation(this.app, {
+          recipient: donation.ownerEntity.email,
+          user: donation.ownerEntity.name,
+          txHash: donation.txHash,
+          donationType: toPledgeAdmin.type, // dac / campaign / milestone
+          donatedToTitle: toPledgeAdmin.admin.title,
+          amount: donation.amount
+        }); 
+
```

Comment:
In the future, maybe we could have a separate notification for a new donation vs a donation that has been delegated/transferred, etc

Reply:
Yes that's the plan (next week I hope). I need to implement all the delegation notifications.
But don't I catch only new donations now?

Reply:
you catch all `transfers`, including...

new donations, committing delegation from dac to project/milestone, transfer from project -> milestone